### PR TITLE
updating LFX Mentorship templates

### PR DESCRIPTION
**Pull request by nate-double-u**
Signed-off-by: Nate W <natew@cncf.io>

### CODE REVIEW

File: lfx-mentorship/2023/01-Mar-May/project_ideas.md [hunk 1]
```diff
@@ -5,12 +5,11 @@
 
 #### Mentorsip project Title
```
Comment (wallyqs) [inline lfx-mentorship/2023/01-Mar-May/project_ideas.md:6]:
```suggestion
#### Mentorship project Title
```

**Comment by aliok** [comment]
LGTM, but the `PROJECT_IDEA_TEMPLATE.md` file is not referenced anywhere. Is there a page where it is written how to kick off a outreach program at CNCF, where that file can be referenced?

File: lfx-mentorship/2023/01-Mar-May/project_ideas.md (hunk 1 above)
Comment (nate-double-u) [reply lfx-mentorship/2023/01-Mar-May/project_ideas.md:6]: Good catch 😅

**Review by jaytiaki — APPROVED** [review]
/lgtm

**Comment by aliok** [comment]
> We are currently working on revising the repo, I expect that issues like orphan pages will be sorted with that.

Sounds good.

> What do you mean "outreach program"?

Like GSoC, LFX etc. 

If there was a document which explains what to do at CNCF side when kicking off GSoC/LFX etc, there could be a link to that template file.

**Comment by nate-double-u** [comment]
> > What do you mean "outreach program"?
> 
> Like GSoC, LFX etc.
> 
> If there was a document which explains what to do at CNCF side when kicking off GSoC/LFX etc, there could be a link to that template file.

Ah, yes, we're working on that too :)

**Comment by aliok** [comment]
> Ah, yes, we're working on that too :)

Cool! I would be happy to have a look if you ping me there

**Comment by nate-double-u** [comment]
@aliok, just to follow up, you can see some of the changes that we're planning in this PR: https://github.com/cncf/mentoring/pull/733. We'd be happy for your feedback!

**Pull request merged**